pkgs/by-name/wh/whph/package.nix

```
@@ -0,0 +1,53 @@
+{
+  lib,
+  fetchFromGitHub,
+  flutterPackages-source,
+  gst_all_1,
+  libunwind,
+  orc,
+  libayatana-appindicator,
+  libnotify,
+}:
+flutterPackages-source.v3_32.buildFlutterApplication rec {
+  pname = "whph";
+  version = "0.14.1";
+
+  src = fetchFromGitHub {
+    owner = "ahmet-cetinkaya";
+    repo = "whph";
+    tag = "v${version}";
+    hash = "sha256-NePIR4FwJ38CWiFuq6uJamKqTy9dRll8C9OHhLlXuZQ=";
+    fetchSubmodules = true;
+    # Remove the large but useless submodule
+    postFetch = "rm -rf $out/src/android";
+  };
+
+  sourceRoot = "${src.name}/src";
+
+  pubspecLock = lib.importJSON ./pubspec.json;
+
+  buildInputs =
+    with gst_all_1;
+    [
+      gstreamer
+      gst-plugins-base
+      gst-plugins-good
+    ]
```

```suggestion
    (with gst_all_1;
    [
      gstreamer
      gst-plugins-base
      gst-plugins-good
    ])
```
proper separation of scopes

---

forgot this one